meshpy/boundary_condition.py

```diff
@@ -80,6 +80,7 @@ def from_dat(cls, bc_key, line, **kwargs):
 
         if bc_key in (
             mpy.bc.dirichlet,
+            mpy.bc.locsys,
```

Comment:
Ordering should match the one in `meshpy/conf.py`. I would put it at the end.

Reply:
Thanks, for catching that.